docs/en/03_How_tos/external_http_requests_with_proxy.md

```diff
@@ -22,63 +22,68 @@ You can look up the implementation details in the `CwpInitialisationFilter` in t
 
 ## Stream-based requests
 
-Stream-based requests however will need extra manual configuration. For example the following will not work, resulting
+Stream-based requests however will need extra manual configuration. For example, the following will not work, resulting
 in a "Connection refused" error:
 
-	:::php
-	echo file_get_contents('http://www.silverstripe.org/blog/rss');
+```php
+echo file_get_contents('http://www.silverstripe.org/blog/rss');
+```
 
 We can modify it to use the proxy. Proxy URL on CWP environments is provided via `SS_OUTBOUND_PROXY` define, and port is
 provided via `SS_OUTBOUND_PROXY_PORT`. We can check for the existence of these so we are not forced to use the proxy on
 the development machine:
 
-	:::php
-	// use proxy if the environment file has a proxy definition
-	if(defined('SS_OUTBOUND_PROXY') && defined('SS_OUTBOUND_PROXY_PORT')) {
-		$context = stream_context_create(array(
-			'http' => array(
-				'proxy' => sprintf('tcp://%s:%s', SS_OUTBOUND_PROXY, SS_OUTBOUND_PROXY_PORT),
-				'request_fulluri' => true
-			)
-		));
-		
-		echo file_get_contents('http://www.silverstripe.org/blog/rss', false, $context);
-	} else {
-		echo file_get_contents('http://www.silverstripe.org/blog/rss');
-	}
+```php
+// use proxy if the environment file has a proxy definition
+if(defined('SS_OUTBOUND_PROXY') && defined('SS_OUTBOUND_PROXY_PORT')) {
```

Comment:
```php
use SilverStripe\Core\Environment;

// use proxy if the environment file has a proxy definition
if (Environment::getEnv('SS_OUTBOUND_PROXY') && Environment::getEnv('SS_OUTBOUND_PROXY_PORT')) {
```

The lines where the constant is used will need to be updated to `Environment::getEnv('SS_OUTBOUND_PROXY')` too (for example)